Tests/Engine/CustomizedRule.tests.ps1

```diff
@@ -9,7 +9,7 @@ if (-not (Test-PSEditionCoreCLR))
 	$null,"Wow6432Node" | ForEach-Object {
 		try
 		{
-			Set-ItemProperty -Name "DisablePromptToUpdateHelp" -Path "HKLM:\SOFTWARE\$($_)\Microsoft\PowerShell" -Value 1 -Force
+			Set-ItemProperty -Name "DisablePromptToUpdateHelp" -Path "HKLM:\SOFTWARE\$($_)\Microsoft\PowerShell" -Value 1 -Force -ErrorAction SilentlyContinue
```

Comment:
Can you please elaborate on the error that you were getting that led you to apply this change? On which environment (platform and ps version) did you see it?

Reply:
running tests in non-elevated windows powershell on Windows 10.

Reply:
Ok, thanks for the info. This does not happen on my machine, I am wondering if this could be caused by some group policy in your environment. Although it seems that this test code should reviewed again and possibly refactored, I think we can still accept your changes to it.